### DISCUSSION TITLE
Remove the /blowfish endpoint?

### DISCUSSION THREAD

**Discussion by TheRealGchen** [nunocoracao/blowfish#421] [Q&A]
Hi all, kind of a noob question but is there a way to remove the need to put `/blowfish` before all endpoints? 

Right now I have to access posts like `<website>/blowfish/posts` but I want to be able to just do `<website>/posts`. Is there a way to un-nest everything from the blowfish level?

**Comment by nunocoracao** [comment]
Need more details to help. By default that path is not there. Not sure how you setup the theme though.

**Reply by nunocoracao** [reply]
@TheRealGchen do you still need help?

**Comment by TheRealGchen** [comment]
No all good! Thanks for the reply. I tried it again after a reinstall and everything worked fine, I think it was just a loose environment variable I had laying around. On Jan 12, 2023, at 5:17 PM, Nuno Coração ***@***.***> wrote:﻿
@TheRealGchen do you still need help?
